MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -969,4 +970,4 @@ protected override string GetCommandLineArguments(string outputPath, EncodingOpt
             ).Trim();
         }
     }
-}
+}
```

Comment:
Oh.. Files should have a newline in the end, if you could set up your editor to do this or add it manually, that would be nice :)

Reply:
Also git can do it for you if you want, that's another option :)

Reply:
I was pretty sure my IDE did this. I just use visual studio 2017 and I even checked it just and it still added the line at the end so I don't know what happened.